code/go/0chain.net/blobbercore/handler/handler.go

```diff
@@ -53,7 +53,7 @@ func SetupHandlers(r *mux.Router) {
 	//object info related apis
 	//r.HandleFunc("/allocation", common.UserRateLimit(common.ToJSONResponse(WithConnection(AllocationHandler))))
 	//r.HandleFunc("/v1/file/meta/{allocation}", common.UserRateLimit(common.ToJSONResponse(WithReadOnlyConnection(FileMetaHandler))))
-	r.HandleFunc("/v1/file/stats/{allocation}", common.UserRateLimit(common.ToJSONResponse(WithReadOnlyConnection(FileStatsHandler))))
+	//r.HandleFunc("/v1/file/stats/{allocation}", common.UserRateLimit(common.ToJSONResponse(WithReadOnlyConnection(FileStatsHandler))))
```

Comment:
is it safe to remove?  I know it is still used on gosdk. https://github.com/0chain/gosdk/blob/826e79c0eb2030c6ba1ada3f0aa7928c67e5d590/zboxcore/zboxutil/http.go#L326

Reply:
see above